src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -114,29 +114,26 @@ public partial class IsolatedStorageFileStream : System.IO.FileStream
         public override bool CanWrite { get { throw null; } }
         // https://github.com/dotnet/corefx/issues/11127
         // [System.ObsoleteAttribute("This property has been deprecated.  Please use IsolatedStorageFileStream's SafeFileHandle property instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
I can't without updating FileStream. I'm going to update the issue mentioned to do so.